crates/bevy_core/src/time/fixed_timestep.rs

```diff
@@ -194,3 +195,79 @@ impl System for FixedTimestep {
         self.internal_system.check_change_tick(change_tick);
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use bevy_ecs::prelude::*;
+    use bevy_utils::Instant;
+    use std::ops::{Add, Mul};
+    use std::time::Duration;
+
+    const LABEL: &str = "test_step";
+
+    #[test]
+    fn test() {
+        let mut world = World::default();
+        let mut time = Time::default();
+        let instance = Instant::now();
+        time.update_with_instant(instance);
+        world.insert_resource(time);
+        world.insert_resource(FixedTimesteps::default());
+        world.insert_resource(0usize);
```

Comment:
I would probably prefer a `Count` newtype for clarity here, but that's just a nit.